requirements.txt

```diff
@@ -6,11 +6,11 @@ gitpython==0.3.2.1
 jinja2
 jsonschema>=2.3.0
 kombu
-mongoengine>=0.8.7,<0.9
+pymongo<3.0
+mongoengine>=0.9
```

Comment:
I'll change the requirements to mongoengine==0.9. I think that should ease the concern here on unbound version of mongoengine with a bound version of pymongo.